php/classes/controllers/class-episode-controller.php

```diff
@@ -274,6 +273,8 @@ public function render_episodes($settings) {
 		);
 
 		$episodes_template_data = apply_filters( 'episode_list_data', $episodes_template_data );
+		
+		wp_enqueue_style( 'ssp-recent-episodes' );
```

Comment:
Could we wrap this in a check, making sure that the style is registered before enqueing.

See https://github.com/CastosHQ/Seriously-Simple-Podcasting/blob/77e714bb77061bafa1b0873d82c2d46fdf36d613/php/classes/controllers/class-players-controller.php#L114 as an example.

Reply:
I don't think this is necessary because these check will also be performed in enqueue: https://developer.wordpress.org/reference/classes/wp_dependencies/enqueue/

Reply:
@goaround thanks, I appreciate that this check is performed in WordPress core, however, I prefer to do such checking in the plugin, that way if the core functionality is ever changed, the plugin check will still take place.

Reply:
As you wish